README.md

```diff
@@ -13,207 +13,111 @@ Istanbul's state of the art command line interface, with support for:
+| `temp-dir` | directory to output raw coverage information to | `String` | `./.nyc_output` |
 
-<img width="500" src="screen.png">
+Configuration can also be provided by `nyc.config.js` if programmed logic is required:
+```js
+'use strict'; 
+const {defaultExclude} = require('test-exclude');
+const isWindows = require('is-windows');
 
-You can use [any reporters that are supported by `istanbul`](https://github.com/istanbuljs/istanbuljs/tree/master/packages/istanbul-reports/lib): `clover`, `cobertura`, `html`, `json-summary`, `json`, `lcov`, `lcovonly`, `none`, `teamcity`, `text-lcov`, `text-summary`, `text`.
+let platformExclude = [
+  isWindows() ? 'lib/posix.js' : 'lib/win32.js'
+];
 
-```bash
-nyc report --reporter=lcov
+module.exports = {
+  exclude: platformExclude.concat(defaultExclude)
+};
 ```
 
-You can find examples of the output for various reporters [here](https://istanbul.js.org/docs/advanced/alternative-reporters).
+### Publish and reuse your nyc configuration
 
-You also have the choice of using a [custom reporter](https://github.com/pedrocarrico/istanbul-reporter-aws-cloudwatch-metrics).
-Install custom reporters as a development dependency and you can use the `--reporter` flag to load and view them:
+nyc allows you to inherit other configurations using the key `extends`.
+As an example, an alternative way to configure nyc for `babel-plugin-istanbul` would be to use the [@istanbuljs/nyc-config-babel preset](https://www.npmjs.com/package/@istanbuljs/nyc-config-babel):
 
-```bash
-nyc report --reporter=<custom-reporter-name>
+```json
+{
+  "nyc": {
+    "extends": "@istanbuljs/nyc-config-babel"
+    //add your custom overrides here
+  }
+}
 ```
 
-## [Producing instrumented source](./docs/instrument.md)
-
-## Setting the project root directory
-
-nyc runs a lot of file system operations relative to the project root directory.
-During startup nyc will look for the *default* project root directory.
-The *default* project root directory is the first directory found that contains a `package.json` file when searching from the current working directory up.
-If nyc fails to find a directory containing a `package.json` file, it will use the current working directory as the *default* project root directory.
-You can change the project root directory with the `--cwd` option.
-
-nyc uses the project root directory when:
- * looking for source files to instrument
- * creating globs for include and exclude rules during file selection
- * loading custom require hooks from the `require` array
-
-nyc may create artefact directories within the project root, such as:
-  * the report directory, `<project-root>/coverage`
-  * the cache directory, `<project-root>/node_modules/.cache/nyc`
-  * the temp directory, `<project-root>/.nyc_output`
+To publish and reuse your own `nyc` configuration, simply create an npm module that exports an `index.json` with your `nyc` config.
```

Comment:
Not sure we want to hit this level of detail here, but the npm module does not have to use `index.json`.  Modules can be written in CJS instead of JSON, https://github.com/istanbuljs/istanbuljs/blob/master/packages/nyc-config-hook-run-in-this-context/index.js for example.

Reply:
ah good catch. thanks, will fix